app/services/clean_unreceived_matches.rb

```diff
@@ -0,0 +1,14 @@
+class CleanUnreceivedMatches
+  def call
```

Comment:
We're using initialize/perform in the other services. Can we keep that API?

Reply:
:+1: